spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
```

Comment:
That's not our job I am afraid. That kind of things should happen in the Hibernate 5 documentation (or a Hibernate 4 to Hibernate 5 migration guide).